src/routes/NavigationPath.tsx

```diff
@@ -13,4 +13,27 @@ export enum NavigationPath {
   // Repository detail
   repositoryDetail = '/repositories/:repositoryName',
   settings = '/organizations/:reponame/settings',
+
+  // Tag Detail
+  tagDetail = '/organizations/:organizationName/:repoName/:tagName',
+}
+
+export function getTagDetailPath(
+  org: string,
+  repo: string,
+  tag: string,
+  arch = null,
+) {
+  let tagPath = NavigationPath.tagDetail.toString();
+  tagPath = tagPath.replace(':organizationName', org);
+  tagPath = tagPath.replace(':repoName', repo);
+  tagPath = tagPath.replace(':tagName', tag);
+  if (arch) {
+    tagPath = tagPath + `?arch=${arch}`;
+  }
+  return tagPath;
+}
+
+export function getDomain() {
+  return process.env.REACT_APP_QUAY_DOMAIN || 'quay.io';
```

Comment:
we cannot do this because we cannot know the quay domain at build time when we are building static assets. The best way is to use a rendered variable when serving the URL and fallback to the env if that variable is not found. There's already a `QUAY_APP_API_URL` can we use that here?

Reply:
`QUAY_APP_API_URL` is also accessed by `process.env`. Do we know the api url ahead of time but not the domain? Would it make more sense to pull the hostname from where the page was rendered from via `window.location.host`?

Reply:
hmm it might make sense if we host it on the same domain as quay but if we don't then we have to figure out something else

Reply:
We may want to investigate another way to make this configurable at runtime.